unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GotoPanelHUD/GotoPanelHUDController.cs

```diff
@@ -0,0 +1,53 @@
+using DCL;
+using DCL.Interface;
+using UnityEngine;
+
+namespace GotoPanel
+{
+    public class GotoPanelHUDController : IHUD
+    {
+        public IGotoPanelHUDView view { get; private set; }
```

Comment:
I think this can be made `private` (or `internal` if you want to access in tests)